File: 2021/nivel 1/torneio_tenis.c
#include <stdio.h>

char partidas[6];

int main(void)
{
  int vitorias = 0;
  for (int i = 0; i < 6; i++)
  {
    scanf(" %c", &partidas[i]);

    if (partidas[i] == 'V')
    {
      vitorias++;
    }
  }
  if (vitorias >= 5)
  {
    printf("1\n");
  } else if (vitorias >= 3)
  {
    printf("2\n");
  } else if (vitorias >= 1)
  {
    printf("3\n");  
  } else {
    printf("-1\n");
  }
}

File: obi/2021/nivel 1/torneio_tenis.py
partidas = []
for _ in range(6):
    partidas.append(input())


vitorias = partidas.count('V')
if vitorias >= 5:
    print(1)
elif vitorias >= 3:
    print(2)
elif vitorias >= 1:
    print(3)
else:
    print(-1)
